intervention_compassion/mappings/intervention_mapping.py

```diff
@@ -127,7 +127,7 @@ class InterventionMapping(OnrampMapping):
         "FocusAreaNumber": None,
         "FundCode": None,
         "FundingStatusCommittedDate": None,
-        "InitiationType": None,
+        "InitiationType": 'InitiationType',
```

Comment:
Your mapping is wrong. The right part should be Odoo valid fields in the model compassion.intervention!